526 separation/training pay

bshyong opened the pull request:
## Description
Reorders sections/screens relating to separation and training pay

## Testing done
local testing

## Screenshots


## Acceptance criteria
- [x] Sections are reordered and modified according to acceptance criteria in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15932

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15932

bshyong committed:
separate and move separation/training pay

bshyong committed:
Merge branch 'master' of github.com:department-of-veterans-affairs/vets-website into 526-page-flow

bshyong committed:
Merge branch 'master' of github.com:department-of-veterans-affairs/vets-website into 526-page-flow

bshyong committed:
imports

bshyong committed:
Merge branch 'master' of github.com:department-of-veterans-affairs/vets-website into 526-page-flow

bshyong committed:
trainingPay unit test

bshyong committed:
add unit tests

bshyong committed:
Merge branch 'master' of github.com:department-of-veterans-affairs/vets-website into 526-page-flow

bshyong commented on src/applications/disability-benefits/all-claims/config/form.js:
```diff
@@ -40,53 +40,54 @@ import { veteranInfoDescription } from '../content/veteranDetails';
 import { disabilitiesOrientation } from '../content/disabilitiesOrientation';
 import { supportingEvidenceOrientation } from '../content/supportingEvidenceOrientation';
 import {
```
alphabetization

cvalarida reviewed (commented):
I wasn't 100% sure what was going on in the ticket, so I asked @djmunoz1 for some clarification. Sounds like we've got one more requirement not outlined in that ticket.

@djmunoz1 When you get a chance, can you update the connected ticket to include the requirement for moving the retirement pay page and renaming it?

markgreenburg commented:
LMK when the page order is updated per latest comments in attached ticket and will be happy to review!

bshyong committed:
Merge branch 'master' of github.com:department-of-veterans-affairs/vets-website into 526-page-flow

bshyong committed:
reorder and rename service pay to retirement pay

bshyong committed:
Merge branch 'master' of github.com:department-of-veterans-affairs/vets-website into 526-page-flow

bshyong committed:
rename retirement pay waiver page

bshyong committed:
fix unit tests

bshyong committed:
Merge branch 'master' of github.com:department-of-veterans-affairs/vets-website into 526-page-flow

bshyong commented:
This has been updated:
- renamed `servicePay` to `retirementPay`
- reordered `retirementPay` per issue https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15932#issuecomment-452083814
- renamed `waive-retirement-pay` to `retirement-pay-waiver`
- modified unit tests to account for renaming

markgreenburg reviewed (approved):
LGTM! Not sure if Chris's page index changes or this should be merged first.

bshyong committed:
resolve merge conflicts
